rs/tests/driver/src/driver/asset_canister.rs

```diff
@@ -13,7 +13,7 @@ use slog::{info, Logger};
 use std::collections::BTreeMap;
 use tokio::task;
 
```

Comment:
Personally I don't like newlines in the import list since it stops the autosort from working.